WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountView.swift

```diff
@@ -19,6 +19,23 @@ struct AddCustomAmountView: View {
 
                         FormattableAmountTextField(viewModel: viewModel.formattableAmountTextFieldViewModel)
 
+                        VStack(alignment: .leading, spacing: Layout.mainVerticalSpacing) {
+                            Text(Localization.percentageInputTitle + " " + viewModel.baseAmountForPercentageString)
+                                .font(.subheadline)
+                                .foregroundColor(Color(.textSubtle))
+
+                            HStack(spacing: Layout.inputFieldVerticalSpacing) {
+                                InputField(placeholder: Localization.percentageInputPlaceholder,
+                                           text: $viewModel.percentage)
```

Comment:
Thanks for the comment @iamgabrielma ! Did you test on device? We use `.keyboardType(.decimalPad)`, so that should avoid these extra characters. If you tested on simulator, with hardware keyboard, then it's possible, but that's not a real case scenario.

Reply:
I did test in simulator! I'm guessing using the paste function in real device would still bring the extra characters over, but is much less likely and definite an edge case.